### PR TITLE
tests: Add positive test for task and mesh shader

### PR DESCRIPTION
**Pull request by ziga-lunarg**
Add a test for task and mesh shader interface with taskNV, mentioned in #2102

### CODE REVIEW

File: tests/vkpositivelayertests.cpp [hunk 1]
```diff
@@ -15594,3 +15594,84 @@ TEST_F(VkPositiveLayerTest, TestShaderInputAndOutputStructComponents) {
...
+
+    AddRequiredExtensions(VK_NV_MESH_SHADER_EXTENSION_NAME);
+    ASSERT_NO_FATAL_FAILURE(InitFramework(m_errorMonitor));
+    if (!AreRequestedExtensionsEnabled()) {
+        printf("%s Extension %s is not supported, skipping test.\n", kSkipPrefix, VK_NV_MESH_SHADER_EXTENSION_NAME);
+        return;
+    }
+    VkPhysicalDeviceMeshShaderFeaturesNV mesh_shader_features = LvlInitStruct<VkPhysicalDeviceMeshShaderFeaturesNV>();
+    mesh_shader_features.meshShader = VK_TRUE;
+    mesh_shader_features.taskShader = VK_TRUE;
```
Comment (ncesario-lunarg) [inline tests/vkpositivelayertests.cpp:15609]: These should probably be initialized to `VK_FALSE`, no?

**Review by ncesario-lunarg — COMMENTED** [review]
LGTM once CI passes!

File: tests/vkpositivelayertests.cpp (hunk 1 above)
Comment (ziga-lunarg) [reply tests/vkpositivelayertests.cpp]: I should call vkGetPhysicalDeviceFeatures2() to check if they are supported, correct?

Comment (ncesario-lunarg) [reply tests/vkpositivelayertests.cpp]: Yes, I believe so. Not sure if those are the only things you need to check for using task/mesh shaders, though.

**Pull request merged**